eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -144,7 +144,7 @@ steps:
     solution: "build\\loc.proj"
     msbuildVersion: "16.0"
     configuration: "$(BuildConfiguration)"
-    msbuildArguments: "/t:AfterBuild"
+    msbuildArguments: "/t:AfterBuild /bl:$(Build.Repository.LocalPath)\\artifacts\\localizationArtifacts\\loc.binlog"
```

Comment:
Was this just for debugging? Did you intend to remove it before merging?

Reply:
Yeah, that was for debugging. Nice catch!